intranet-webapp/MediaLibrary.Intranet.Web/Controllers/WebApiController.cs

```diff
@@ -133,20 +133,32 @@ public async Task<IActionResult> UpdateMediaItem(string id, [FromBody] MediaItem
             }
         }
 
-        [HttpDelete("/api/media/{name}", Name = nameof(DeleteMediaFile))]
-        public async Task<IActionResult> DeleteMediaFile(string name)
+        [HttpDelete("/api/media/{id}", Name = nameof(DeleteMediaFile))]
+        public async Task<IActionResult> DeleteMediaFile(string id, [FromBody] String itemName)
```

Comment:
Please rename function to `DeleteMediaItem` to be more accurate

```suggestion
        [HttpDelete("/api/media/{id}", Name = nameof(DeleteMediaItem))]
        public async Task<IActionResult> DeleteMediaItem(string id, [FromBody] String itemName)
```